line/nodes/conversation_context.py

```diff
@@ -64,3 +59,28 @@ def get_event_count(self) -> int:
     def add_metadata(self, key: str, value: Any) -> None:
         """Add metadata for specialized processing."""
         self.metadata[key] = value
+
+    def get_committed_turns(self) -> list[Union[UserTranscriptionReceived, AgentResponse]]:
+        """Get all committed transcript messages from the conversation events."""
+        pending_turns = []
+        committed_turns = []
+        for event in self.events:
+            if isinstance(event, UserTranscriptionReceived):
+                committed_turns.append(event)
```

Comment:
Is UserTranscriptionReceived always aggregated, or can it be streamed transcriptions?